src/engraving/libmscore/factory.cpp

```diff
@@ -163,6 +164,7 @@ static const ElementName elementNames[] = {
     { ElementType::STAFFTYPE_CHANGE,     "StaffTypeChange",      QT_TRANSLATE_NOOP("elementName", "Staff type change") },
     { ElementType::HARMONY,              "Harmony",              QT_TRANSLATE_NOOP("elementName", "Chord symbol") },
     { ElementType::FRET_DIAGRAM,         "FretDiagram",          QT_TRANSLATE_NOOP("elementName", "Fretboard diagram") },
+    { ElementType::HARP_DIAGRAM,         "HarpPedalDiagram",          QT_TRANSLATE_NOOP("elementName", "Harp Pedal Diagram") },
```

Comment:
Minor thing: for ui texts, we use "Sentence case" instead of "Title Case". So this one would be "Harp pedal diagram".